src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -661,6 +664,45 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         e.Handled(handled);
     }
 
+    // Method Description:
+    // - Some key events cannot be handled (e.g. AltGr combinations) and are
+    //   delegated to the character handler. Just like with _TrySendKeyEvent(),
+    //   the character handler counts on us though to:
+    // - Clears the current selection.
+    // - Makes the cursor briefly visible during typing.
+    void TermControl::_HandleVoidKeyEvent()

```

Comment:
nit: I think the name of this method could probably be better - the important meat of what this function is trying to accomplish isn't the sending of the void keypress, it's the dismissing of the selection and the cursor visibility.

Reply:
Do you have a suggestion for the function name? I can't come up with anything. 😕
(If we had something different to `SendKeyEvent` - like e.g. `GenerateKeyEvent`, which returns an optional input sequence - we could more comfortably abstract "reset selection and cursor" I suppose. That way you could move the "reset selection and cursor" handling to the more correct WM_CHAR handler right?)

Reply:
The fact that `_TrySendKeyEvent()` supports a `vkey == 0` seemed kinda hacky to me and I wanted to hide this hacky fact even from calls within the same class. So I created a hacky-wrapper function and prevent other methods from being "spoiled". 😄
Do you want me to inline this function?